README.md

```diff
@@ -77,6 +79,48 @@ Here's an example of using it in nixos configuration.
 }
 ```
 
+### Example: Using `devShell` and `nix develop`
+
+Running `nix develop` will create a shell with the default nightly Rust toolchain installed:
+
+```nix
+{
+  description = "A devShell example";
+
+  inputs = {
+    nixpkgs.url      = "github:nixos/nixpkgs/nixos-unstable";
+    rust-overlay.url = "github:oxalica/rust-overlay";
+    flake-utils.url  = "github:numtide/flake-utils";
+  };
+
+  outputs = { self, nixpkgs, rust-overlay, flake-utils, ... }:
+    flake-utils.lib.eachDefaultSystem (system:
+      let 
+        overlays = [ (import rust-overlay) ];
+        pkgs = import nixpkgs {
+          inherit system overlays;
+        };
+      in
+      {
+        devShell = pkgs.mkShell {
+        	buildInputs = [
+          	pkgs.openssl
```

Comment:
Please use spaces instead of tabs.

Reply:
Gvim on my windows box didn't have `expandtab` set! Fixed